schema/deploy/mutations/save_product_mutation_chain.sql

```diff
@@ -24,7 +24,8 @@ begin
   update ggircs_portal.product
   set state = 'deprecated',
       deleted_at = now(),
-      deleted_by = 'Admin' --TODO: Should this be included in the triggers?
+      --TODO: Should this be included in the triggers?
```

Comment:
This TODO is outdated. deleted_at & deleted_by are set by triggers. But I will just make a note to remove it in my next PR,